src/util/migrate.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <filesystem>
+
+namespace big
+{
+    void do_migration(const std::filesystem::path& current_dir)
+    {
+        std::filesystem::path old_dir = std::getenv("appdata");
+        old_dir /= "BigBaseV2";
+
+        if (!exists(current_dir))
+        {
+            LOG(INFO) << "YimMenu folder does not exist, migrating to new folder.";
+            std::filesystem::copy(old_dir, current_dir, std::filesystem::copy_options::recursive);
```

Comment:
It shouldn't try to do this if neither BigBaseV2 or YimMenu folders exist.

Just a nitpick as it seems the only "problem" it would cause is an extra log line for new users.